packages/components/popover/stories/Popover.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+
+import { Popover, PopoverTrigger, PopoverContent } from '../src/.';
+import type { PopoverProps } from '../src/.';
+import { Button } from '@contentful/f36-button';
+import { Text } from '@contentful/f36-typography';
+import { Box } from '@contentful/f36-core';
+
+export default {
+  component: Popover,
+  title: 'Components/Popover',
+  argTypes: {
+    isOpen: { control: { disable: true } },
+  },
+} as Meta;
+
+export const Basic: Story<PopoverProps> = (args) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Popover {...args} isOpen={isOpen} onClose={() => setIsOpen(false)}>
+      <PopoverTrigger>
+        <Button onClick={() => setIsOpen(!isOpen)}>Toggle</Button>
```

Comment:
How is this button not blue in storybook?
Is `secondary` the default value of a button now?

Reply:
Yeah, the Button has the secondary as the default value for the variant

Reply:
And it was aligned with our designers, right @massao ?